experimental/language-generation/README.md

```diff
@@ -1,6 +1,6 @@
 # Language Generation ***_[PREVIEW]_***
 
-> See [here](#Change-Log) for what's new in 4.6 PREVIEW release.
+> See [here](#Change-Log) for what's new in 4.7 PREVIEW release.
```

Comment:
4.6.0.preview2

Reply:
Updated to 4.6 Preview 2 per shiproom discussion.